src/components/Modal.js

```diff
@@ -162,34 +160,35 @@ const Modal = React.createClass({
   },
 
   render () {
-    if (this.props.isOpen)
-      return (
-        <div className='mx-modal' style={[styles.scrim, this.props.isRelative && styles.relative]}>
-          <div className='mx-modal-scrim' onClick={this.props.onRequestClose} style={[styles.scrim, styles.overlay, this.props.isRelative && styles.relative]}></div>
-          <div
-            className='mx-modal-container'
-            style={[styles.container, this.props.style]}
-          >
-            <Icon
-              className='mx-modal-close'
-              onClick={this.props.onRequestClose}
-              size={24}
-              style={styles.close}
-              type='close-solid'
-            />
-            {this._renderTitleBar()}
-            <div className='mx-modal-content' style={[styles.content, this.props.contentStyle]}>
-              {this.props.children}
-              {this._renderTooltip()}
-            </div>
-            {this._renderFooter()}
+    /*eslint-disable */
+    if (this.props.isOpen) {
+      console.warn('WARNING: The prop "isOpen" is depracated in this version of the component. Please handle Modal opening from its parent.');
+    }
```

Comment:
So this wouldn't only work if `this.props.isOpen` is set to true. We want to check to see if the prop is defined. So something like this would be better: `if (this.props.isOpen !== null) {`.

We should also move this into the `componentDidMount` or the `componentWillMount` method.